x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/management/monitor_list_table/columns.tsx

```diff
@@ -131,16 +128,64 @@ export function useMonitorListColumns({
       name: i18n.translate('xpack.synthetics.management.monitorList.actions', {
         defaultMessage: 'Actions',
       }),
-      render: (fields: EncryptedSyntheticsSavedMonitor) => (
-        <Actions
-          euiTheme={euiTheme}
-          configId={fields[ConfigKey.CONFIG_ID]}
-          name={fields[ConfigKey.NAME]}
-          canEditSynthetics={canEditSynthetics}
-          reloadPage={reloadPage}
-          isProjectMonitor={fields[ConfigKey.MONITOR_SOURCE_TYPE] === SourceType.PROJECT}
-        />
-      ),
+      actions: [
```

Comment:
why move this out of it's own component?

Reply:
instead of a custom component, it has been refactored to use standard actions from EuiBasicTable